cpp/include/cuvs/neighbors/ivf_flat.hpp

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) 2024, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <raft_runtime/neighbors/ivf_flat.hpp>
```

Comment:
The intent of these header files is a lot different than the intent of the header files in RAFT. These should be self-contained and not just wrappers around `raft_runtime`.  Essentially what we want is to move `raft_runtime` over to cuVS.